src/gateways/govNotifyApiGateway.ts

```diff
@@ -11,7 +11,14 @@ export interface IGovNotifyGateway {
 export class GovNotifyGateway implements IGovNotifyGateway {
   private api;
   constructor(apiKey: string) {
-    this.api = new NotifyClient(apiKey);
+    if (!apiKey) {
+      // eslint-disable-next-line no-console
+      console.log(
```

Comment:
Is this going to stop the client from failing as it's going to call `this.api.sendEmail` on the `sendEmail` method, unless you put a guard on checking whether the api client has been initialised or not

Reply:
Yeah that's fixed it because (I think) the try/catch captures failures where `this.api` is not initialised.  The failure is now on `authGateway` for similar reasons